### DISCUSSION TITLE
SOLWEIG as design tool

### DISCUSSION THREAD

**Discussion by NizGuarie** [UMEP-dev/UMEP#354] [Q&A]
Hi all,
I'm new to UMEP and relatively new to QGIS, so apologies in advance if this might appear a trivial question.
I managed to apply the simulation process described in the SOLWEIG tutorials to a personal project I'm carrying out, by using a local raster data of Zurich. This has allowed me to analyse the current conditions of outdoor comfort. However I was wandering if there is any way to use SOLWEIG as design tool and include 3D blocks of possible buildings with different geometries (as shapefile) and assess the effect of these building blocks on the outdoor climate.
Thanks a lot in advance.
Niz

**Comment by biglimp** [comment]
Hi Niz, welcome to the community. UMEP make predominantly use of raster data. I can recommend that you take a look at the DSMGenerator in UMEP. That tool might help you convert vector to raster.
